userdocs/profiles.dev/docs/intro.md

```diff
@@ -18,7 +18,7 @@ Thank you!
 
 ## What are Profiles?
 
-In short, a profile is a Kubernetes package. Much as you would expect from another package
+In short, Profiles is a Kubernetes package manager. Much as you would expect from another package
```

Comment:
technically, pctl is the package manager, and Profiles is a concept of package management

Reply:
yeah I know I was kinda struggling with it. Kinda tweaked it a little with your suggestion, imo this section should be related to `Profiles` since we are explaining `profile` in another section. wdyt looks better?

Reply:
yeh it's really tough, because people will read `Profiles` and immediately hear the plural of "more than one profile", so it will always be jarring to read `Profiles is...`.

If we go with the change you just added, logically it would make sense to change the heading to `What is Profiles?` which my brain just will not compute 😂 . This is why I originally had this bit talk about profiles (as in multiple units) rather than the concept.

So I think we have two options:
- On all user facing docs we don't use the word "Profiles" as a concept, just talk about "profiles" as an synonym for packages and how you use `pctl` to install them etc
- We keep using "Profiles" and "profiles", and in this bit here we remove the subheading (basically removing the `is/are` trap) and take care to avoid it in future.

Having written all that I am leaning to the former I think 🤔

Reply:
this seems to be going in circles, so I am going to put it back to what it was. Bianca suggested I should get people involved from product and/or marketing so we can come up with content for this, also we need some suggestions/content to showcase for the homepage. For now, I am adding few styling changes pointed out by Bianca and we can wrap this up :)